frontend/src/components/worksheets/BundleDetail/MainContent.jsx

```diff
@@ -40,73 +49,81 @@ class MainContent extends React.Component<
 
 		return (
             <div className={ classes.outter }>
-                <div className={ `${ classes.stateBox } ${ classes[stateSpecClass] }`}>
-                    { bundleState }
-                </div>
-    			<Grid container classes={ { container: classes.container } } spacing={16}>
-                    { /** Run bundle specific components =========================================================== */}
-                    { isRunBundle &&
-                        <Grid item xs={12}>  
-                            <CopyToClipboard
-                                text={ bundleInfo.command }
-                            >
-                                <div
-                                    className={ `${ classes.row } ${ classes.command }` }
+    			<Grid container>    

```

Comment:
shows stdout&stderror toggler when these two messages exist, fileBrowser toggler is always shown